source/MRMesh/MRCNCMachineSettings.h

```diff
@@ -19,12 +21,17 @@ class CNCMachineSettings
         C
     };
     using RotationAxesOrder = std::vector<RotationAxisName>;
+    using RotationLimits = std::optional<Vector2f>;
 
     static int getAxesCount() { return int( RotationAxisName::C ) + 1; }
 
     // rotationAxis length will be more then 0.01
     MRMESH_API void setRotationAxis( RotationAxisName paramName, const Vector3f& rotationAxis );
     MRMESH_API const Vector3f& getRotationAxis( RotationAxisName paramName ) const;
+    // rotationLimits = {min, max}
+    // valid range -180 <= min < max <= 180
```

Comment:
looks strict mb make defaut [-360 360] ?

Reply:
i think it's unlikely